tvbwidgets/core/pse/parameters.py

```diff
@@ -369,6 +382,6 @@ def launch_local_param(param1, param2, x_values, y_values, metrics, file_name):
     n = len(sys.argv[5])
     metrics = sys.argv[5][1:n - 1].split(', ')
     file_name = sys.argv[6]
+    update_progress = sys.argv[7]
```

Comment:
We can not serialize a function here. So we should better leave the HPC part for a separate task altogether